PayPalDataCollector/src/main/java/com/paypal/android/paypaldatacollector/PayPalDataCollectorEnvironment.kt

```diff
@@ -0,0 +1,13 @@
+package com.paypal.android.paypaldatacollector
+
+enum class PayPalDataCollectorEnvironment {
```

Comment:
doc strings for this class

Reply:
Actually would we expect merchants to use this? Might be a good time to introduce those [@RestrictTo](https://developer.android.com/studio/write/annotations#libraries) annotations.

Reply:
@sshropshire we have to expose the environment somehow. Exposing the `Magnes` `Enviornment` class directly seems to me that defeats the purpose of this module, that essentially wraps `Magnes`.
How would you use the `@RestrictTo` in this case?

Reply:
If we can reduce the amount of documentation we have to maintain I think it's a win. `RestrictTo` calls out to merchants via the linter that this is library code, and shouldn't be used by the merchant. It's still technically accessible, but with a warning.

Reply:
This may be a broader conversation though. We would need to confirm that NXO uses the same `groupId` that we do for our artifacts. Since Kotlin doesn't have a `package-private` feature like Java, I think it's in our best interest to draw the line between library code and merchant facing API. We'll have to do the same thing on iOS.

Reply:
@sshropshire ok gotcha. But it seems to me that this doesnt apply here as this class is part of the merchant facing API.